doc/security/hardening-tool.rst

```diff
@@ -0,0 +1,44 @@
+.. _hardening:
+
+Hardening Tool
+##############
+
+Zephyr contains several options features that makes the overall system
+more secure. As we take advantage of hardware features, many of these
+options are platform specific and besides it, some of them are not
+known by developers.
+
+To address this problem, we provide a tool that helps to check an
+application Kconfig option list against a list of hardening
+preferences defined by the **Security Group**. The tool is capable of
+understand what is the build target and suggest specific
+recommendations to it.
+
+Usage
+*****
+
+After configure your application, cd to the build folder and:
```

Comment:
after configuration of your application, change directory to the build folder...